src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class ShooterSubsystem extends SubsystemBase {
+
+  private final CANSparkMax shooterMotor;
+  boolean readyToFire = true; // Take in readiness state from ShooterTrigger?
```

Comment:
There are several parts to being "readyToFire"  

One is having a note in the trigger, another is having the turret in position, another is the shooter rollers are at the correct speed.

Let's remove this, and instead provide a trigger that indicates whether the shooter rollers are the correct speed.

Reply:
Resolved with upcoming commit